_docs/quickstart.md

```diff
@@ -8,7 +8,7 @@ For the impatient, here's how to do function tracing with Frida:
 
 {% highlight bash %}
 ~ $ sudo pip install frida
-~ $ frida-trace -i "recv*" -i "read*" Twitter
+~ $ frida-trace -i "recv*" -i "read*" com.twitter.android
```

Comment:
Perhaps we should leave this as “Twitter” so it doesn't sound like an Android-specific example? (You'd typically use the `-U` switch if the process was on a connected Android device.)